pkg/smokescreen/metrics/metrics.go

```diff
@@ -56,6 +56,9 @@ type MetricsClientInterface interface {
 	AddMetricTags(string, []string) error
 	Incr(string, float64) error
 	IncrWithTags(string, []string, float64) error
+	Gauge(string, float64, float64) error
```

Comment:
No `GaugeWithTags`? Just wondering for consistency's sake.

Reply:
It feels a little weird to me to add dead code just for the sake of having it there, especially since we're already covering such a limited set of the statsd client interface with this API (and because these methods are pretty trivial to implement, I don't think it'll get it in the way of anyone who wants to add tags to their gauge down the line)